custom_components/waste_collection_schedule/waste_collection_schedule/source/hvcgroep_nl.py

```diff
@@ -169,14 +306,25 @@ def fetch(self):
             raise Exception("no data found for this address")
 
         bag_id = data[0]["bagid"]
+        if len(data) > 1 and self.house_letter and self.postfix:
+            _LOGGER.info(f"Checking {self.house_letter} {self.postfix}")
+            for address in data:
+                if (
+                    address["huisletter"] == self.house_letter
+                    and address["toevoeging"] == self.postfix
```

Comment:
Hi, I happened to configure the plugin yesterday with this service and came across this pull request.

Where does postfix come from? I see 3 names:
- addition (in docs)
- suffix on line 289 and 294
- postfix here on line >309

It is also possible to have no house letter (huisletter) but to have a postfix/addition/suffix (toevoeging).

The data[0]["bagid"] should not make this a breaking change but this seems incorrect.

Reply:
good catch. I couldn't decide on the right name so changed it around a few times and probably didn't copy the last version of my code into my local git repo after some testing 😅